tests/functional/conftest.py

```diff
@@ -91,12 +83,6 @@ def factories(db_session):
     factories_common.set_session(None)
 
 
-@pytest.fixture(scope="session", autouse=True)
-def init_db(db_engine):
-    authority = TEST_SETTINGS["h.authority"]
-    db.init(db_engine, should_drop=True, should_create=True, authority=authority)
```

Comment:
These are standard, simple versions of these DB fixtures, the same as the ones in Via. They're in `tests/conftest.py` where they can be shared by both the unit and functional tests. h had its own versions of these and had different versions for the unit and functional tests.